mvlearn/semi_supervised/ctclassifier.py

```diff
@@ -271,14 +271,14 @@ def fit(self, Xs, y):
         if self.n_classes > 1:
 
             # if both p & n are none, set as ratio of one class to the other
-            if (self.p_ is None and self.n_ is None):
```

Comment:
also I would avoid single letter attributes. Use n_samples_ and n_features_ ?

Reply:
Changed them to ``positive_samples`` and ``negative_samples``